enterprise/api_client/lms.py

```diff
@@ -398,6 +398,65 @@ def _get_results(self, identity_provider, param_name, param_value, result_field_
         return None
 
 
+class ThirdPartyAuthApiClientJwt(JwtLmsApiClient):
+    """
+    Object builds an API client to make calls to the Third Party Auth API.
+    """
+
+    API_BASE_URL = settings.LMS_INTERNAL_ROOT_URL + '/api/third_party_auth/v0/'
+
+    @JwtLmsApiClient.refresh_token
+    def get_remote_id(self, identity_provider, username):
+        """
+        Retrieve the remote identifier for the given username.
+
+        Args:
+        * ``identity_provider`` (str): identifier slug for the third-party authentication service used during SSO.
+        * ``username`` (str): The username ID identifying the user for which to retrieve the remote name.
+
+        Returns:
+            string or None: the remote name of the given user.  None if not found.
+        """
+        return self._get_results(identity_provider, 'username', username, 'remote_id')
```

Comment:
to make it more readable can we call like this
`self._get_results(identity_provider, 'remote_id', query_parms={'username': username} )`